lang/cs/Org.Apache.REEF.Tang/Implementations/ClassHierarchy/ClassHierarchyImpl.cs

```diff
@@ -363,6 +388,44 @@ public INode GetNode(string fullName)
             return this.GetNode(t);
         }
 
+        public INode GetNode(string fullName, string aliasLanguage)
```

Comment:
It implements public interface so has to be public.  As I mentioned in previous comments, the we should make entire class hierarchy interface as internal. We should do it in a separate Jira.